cmd/triggerRun/testdata/trigger.yaml

```diff
@@ -1,10 +1,12 @@
-apiVersion: tekton.dev/v1alpha1
+apiVersion: triggers.tekton.dev/v1alpha1
 kind: Trigger
 metadata:
   name: trigger-run
+  namespace: "default"
```

Comment:
I think if we don't add also it will take the `default` namespace

Reply:
I think this is due to the lines here: https://github.com/tektoncd/triggers/pull/695/files#diff-3ad34365d6cb2174d0f7c75aacea5476R213

we are using `tri.namespace` directly -> which means that if you don't specify a namespace it will be an empty string. Instead we could do a simple check and use "default" as the namespace if `tri.namespace` is an empty string